VERSION

```diff
@@ -1 +1 @@
-1.12.0
+1.12.1
```

Comment:
technically this change is not backwards compatible because consumers now need to explicitly ask for RPM output. However, this is a very recent change so we can assume that it is not used (and we are the consumers of this repo).

unless someone from @conjurinc/conjur-core-team thinks we need to change this to `2.0.0` i'm ok with this version.

Reply:
also - @tzheleznyak can you please add a CHANGELOG entry? this is a customer-facing change (the customer is internal in this case but still).

Reply:
regarding the above - do we have an issue that we can link to in the CHANGELOG? if not, please create one.

Reply:
Dont you think it is too little change to change number main number of version?

Reply:
Changed ChangeLog

Reply:
We use [SemVer](https://semver.org/) in our versioning process. Would you say that this change fits sections 1, 2 or 3?

Reply:
Added Issue

Reply:
i would change it to `1.13.0` before changing to `1.12.2` as this is not a bug fix rather than a functionality change. But because it is not backwards compatible (users who ran this got RPMs whereas after this change they won't) then i think a MAJOR change is more fitting.

Reply:
Changed to 2.0.0